src/main/xar-resources/data/configuration.xml

```diff
@@ -1350,12 +1405,12 @@ uri="http://exist-db.org/xquery/file" /&gt;</synopsis>
                     <synopsis language="xquery">import module namespace file="http://exist-db.org/xquery/file";</synopsis>
                     <para>Instead of providing a Java class, one can also specify a src URI which
                         must point to the XQuery source code of the module, e.g.:</para>
-                    <synopsis language="xml">&lt;module src="resource:org/exist/xquery/lib/json.xq" 
+                    <synopsis language="xml">&lt;module src="resource:org/exist/xquery/lib/json.xq"
```

Comment:
Might I suggest replacing this example module (which I wouldn't suggest anyone use now that we have proper support for JSON in XQuery) with `<module uri="http://exist-db.org/xquery/kwic" src="resource:org/exist/xquery/lib/kwic.xql"/>`?